src/hooks/jobs/useFeedbackParser.ts

```
@@ -0,0 +1,24 @@
+
+import { InterviewFeedback } from "@/lib/types";
+
+export const parseFeedback = (feedbackJson: any): InterviewFeedback | undefined => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)